src/west/main.py

```diff
@@ -26,9 +26,8 @@
 from west.commands.build import Build
 from west.commands.flash import Flash
 from west.commands.debug import Debug, DebugServer, Attach
-from west.commands.project import List, Clone, Fetch, Pull, Rebase, Branch, \
```

Comment:
I just noticed the previous patch breaks `git bisect` - the removed commands should have been removed from the import list there, not here. But we're in a hurry so let's not fix this if there's not time.

Reply:
Removal of import is now moved to correct commit, although it doesn't make it bisectable as tests are still failing, due to `west clone` now missing, and `west update is not available until next commit.`

Reorderring commits however, will require updating test code which will anyway be removed due to commands (`clone/fetch/rebase/etc.`) being removed.